### PR TITLE
Use both names

### PR DESCRIPTION
**Pull request by mier85**
So there are actually (at least) two ways to marshal a proto message...
If you have 

```
message Example {
    string my_example = 1;
}
```

Depending on the options you chose for the marshaller, it either marshalls to
``` 
{"my_example":"boo" }
``` 

or

``` 
{"myExample":"boo"}
```

So if we want to use json schema to validate messages, we need to consider both ways, as excess fields are just ignored. This PRs simply adds the second possible name

### CODE REVIEW

**Comment by chrusty** [comment]
Hi @mier85.

I think i'd prefer to have this behind an command-line flag, otherwise everyones JSONSchemas will be twice as big!

**Comment by mier85** [comment]
@chrusty I understand your concerns. However, then it'd have to be configured which of the name is actually used for the default option GetJsonName sounds like the right function for the json schema. 

But since both is actually allowed in proto, if only one of the field names is allowed, it makes it impossible to actually validate an event without letting 50% of invalid events go through. So I think, even if we have to double the file size, it has to be done?

**Comment by chrusty** [comment]
@mier85 Just make a new parameter called `proto_and_json_fieldnames` which enables this behaviour.

**Comment by mier85** [comment]
So I made it optional and kept the default values as they are for now to avoid breaking changes

**Comment by chrusty** [comment]
Thanks for this, looks good. How about a test for the suite?

**Comment by chrusty** [comment]
@mier85 I'd like to merge your PR before tacking this next one: https://github.com/chrusty/protoc-gen-jsonschema/pull/14

I'll make sure your functionality makes it in-tact, this is why I'm pressing for tests for each feature.

**Comment by mier85** [comment]
i added a test

File: main_test.go [hunk 1]
```diff
@@ -195,3 +209,7 @@ func configureSampleProtos() {
 	}
 
 }
+
+func TestUseProtoAndJSONFieldnames(t *testing.T) {
```
Comment (chrusty) [inline main_test.go]: I'm glad we have this

**Pull request merged**